src/Resources/ShipmentOptions.php

```diff
@@ -58,6 +61,24 @@ public function setDescriptionAttribute(string $value): void
         $this->label_description = $value;
     }
 
+    /**
+     * Set Insurance amount. Only applicable for package_type = 1 (package)
+     * @param  int  $value
+     * @param  string  $currency
+     * @return $this
+     */
+    public function setInsurance(int $value, string $currency = 'EUR'): self
+    {
+        if ($this->package_type === 1) {
```

Comment:
Please use a constant here: `PackageType::PACKAGE`

Reply:
Constants aren't used in shippingOptions, wanted to make a new PR in which all constants are applied

Reply:
You can always import that. Noticed myself constants weren't used in all places. Fixed that in this [commit](https://github.com/mvdnbrk/myparcel-php-api/commit/ef759e152325f7ea766b86393da2d151507d8c9f). Constant can be used now.